src/App.css

```diff
@@ -2,17 +2,24 @@
   max-width: 1280px;
   margin: 0 auto;
   padding: 2rem;
-  text-align: center;
+}
+
+body {
+  background-color: #f8f8f8;
+  font-family: "Open Sans", sans-serif;
+  font-size: 14px;
```

Comment:
Don't give a font-size at the body level.

Reply:
Sure Sir I can remove it..